### PR TITLE
Bug 1592855 added step for labeling default project to managing networking

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1592855

Straight-forward issue.

@openshift/team-documentation PTAL

### CODE REVIEW

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -1417,6 +1418,21 @@ new project is created. To do this:
 . Create a custom project template and configure the master to use it, as
 described in
 xref:../admin_guide/managing_projects.adoc#modifying-the-template-for-new-projects[Modifying the Template for New Projects].
+
+. Label the `default` project with the `default` label:
++
+[IMPORTANT]
+====
+Skip this step if you have labeled the default project with the `default` label
```
Comment (ahardin-rh) [inline admin_guide/managing_networking.adoc]: If you labeled the default project with the `default` label in a previous procedure, then skip this step.

**Comment by ahardin-rh** [comment]
@bfallonf Just one suggestion from me :fireworks:

**Comment by bfallonf** [comment]
Thanks @ahardin-rh ! Changed to suggestion. I'll merge :cactus:

**Pull request merged**